### ISSUE TITLE
[Bug] Quick fix to correct sequential playback not working properly.

### ISSUE THREAD

**Issue by ob89** [Tikipeter/tikipeter.github.io#190]
Hi,

I think I found small issue in RD sources list. 
Tested on latest fen light version 2.0.0.4. 
My sorting in settings is: By quality, size, provider. 
I am using real debrid.

I looked for a movie with a lot of uncached sources and found the movie “Samurai Princess”.
There is only 1 link that is working and it is the 1GB 1080p one.
When I start with 720p link it jumps back to 1080p after 5 attempts of the 720p category, instead of going through all the 40 uncached links in 720p + SD categories - and only then it should try the 1080p links.
(By order of sources list shown)

Hoping you could fix this before your well deserved retirement.
Thank you for all this years and by the way I am still using Fen and love it. 
I only tested it on fen light to make sure it does not only affect us Fen users who still prefer better support for other langauge than english.
Once again thank you for all this years for Fen and Fen light.

**Comment by ob89** [comment]
Just tested it with 2.0.0.6 and it is still jumping from resolution to resolution and from different file sizes not in the correct order 
and only trying around 8 times instead of going on all the sources.
Do you need more details? Why did you close this issue?

**Comment by bbviking** [comment]
What results sorting criteria are you using?

And do you have any 'custom autoplay sort to top' criteria specified?

On my end the sources attempt to play in the order that I would expect them to

**Comment by ob89** [comment]
I did a clean install.
I installed fen light and did not touch the setting.
The default looks like resoultion and than size.
I pressed the first 720p link and all of a sudden it jumps to 1080p.
In the movie "Samurai Princess" there are 40 uncached links most of them 720p and sd.
I think there are 2 1080p and only 1 of them is working.
There is only 1 1080p link that is working from all the sources in "Samurai Princess" everthing else in uncached.
As I said completly clean install and no change in settings.

I am using Fen becasue I need the plot in different language and only install Fen Light for testing this problem.
